doc/release/RELEASE-PROTOCOL.md

```diff
@@ -153,7 +154,7 @@ Make sure that:
 
 1. To enhance communication within the team, the responsible person for the release must notify all stakeholders about the release two working days before its scheduled date and in 1-2 hours to prevent any actions on the environment on which the deployment is carried out.
 2. The responsible person have to take db snapshot manually via Amazon RDS in the `Snapshots` tab with name `env-db-date` (examples: `stg-db-05-18-2024` and `prd-db-05-18-2024`).
-3. On the designated time and day, the responsible person runs the `Release [Deploy]` workflow for the sandbox and production environments from the release branch. They need to fill in the full release tag version (`X.Y.Z`) and choose the environment.  
+3. On the designated time and day, the responsible person runs the `Release [Deploy]` workflow for the sandbox and production environments from the release branch. They need to fill in the full release tag version (`X.Y.Z`) and choose the environment. If need to clear OpenSearch indexes during deployment, they must select the Clear OpenSearch indexes checkbox.
```

Comment:
> If need to clear OpenSearch indexes during deployment, they must select the Clear OpenSearch indexes checkbox.

I think you missed **you**. If **you** need to clear ....

Reply:
done